docs/cli/graphql-transformer/directives.md

```diff
@@ -476,60 +476,66 @@ type Post
+In this schema, only the owner of the object has the authorization to perform read (`getTodo` and `listTodos`) and update operations (`updateTodo` and `deleteTodo`) on the owner created object. This prevents any other owner (any user other than the creator of the object) to update/delete some other owner's object.
+
 Here's a truth table for the above-mentioned schema. In the table below `other` refers to any user other than the creator or owner of the object.
 
 |       | getTodo | listTodos | createTodo | updateTodo | deleteTodo |
 |:------|:-------:|:---------:|:----------:|:----------:|:----------:|
 | owner |    ✅   |     ✅    |     ✅     |      ✅   |     ✅     |
-| other |    ❌   |     ❌    |     ✅     |      ✅   |     ✅     |
+| other |    ❌   |     ❌    |     ✅     |      ❌   |     ❌     |
 
-If you want to prevent updates and deletes operations, you would need to modify the @auth rule to explicitly include the `update` and `delete` operation and your schema should look like the following:
+Next, let's say that we wanted wanted to modify the schema to allow anyone to create, read, and update, but only the owner to be able to delete.
 
 ```graphql
 type Todo @model
-  @auth(rules: [{ allow: owner, operations: [read, update, delete] }]) {
+  @auth(rules: [{ allow: owner, operations: [create, delete] }]) {
   id: ID!
   updatedAt: AWSDateTime!
   content: String!
 }
 ```
 
+In this schema, only the owner of the object has the authorization to perform delete operations on the owner created object, but anyone can read or update them.
+
 Here's a truth table for the above-mentioned schema. In the table below `other` refers to any user other than the creator or owner of the object.
 
 |       | getTodo | listTodos | createTodo | updateTodo | deleteTodo |
-|:------|:-------:|:--------:|:----------:|:----------:|:----------:|
-| owner |    ✅    |     ✅    |      ✅     |      ✅     |      ✅     |
-| other |    ❌    |     ❌    |      ✅     |      ❌     |      ❌     |
+|:------|:-------:|:---------:|:----------:|:----------:|:----------:|
+| owner |    ✅   |     ✅    |     ✅     |      ✅   |     ✅     |
+| other |    ✅   |     ✅    |     ✅     |      ✅   |     ❌     |
 
 
-> **Note:** Specifying `@auth(rules: [{ allow: owner, operations: [create]}])` still allows anyone who has access to your API to create records (as shown in the above truth table). However, including this is necessary when specifying other owner auth rules to ensure that the owner is stored with the record so it can be verified on subsequent requests.
+> **Note:** Specifying `@auth(rules: [{ allow: owner, operations: [create]}])` allows any signed in user to create records (as shown in the above truth table). Including this is necessary when specifying other owner auth rules to ensure that the owner is stored with the record so it can be verified on subsequent requests.
```

Comment:
Hmmm, I'm unclear on what this means. This particular incantation feels very important to me and maybe should be spelled out completely and explicitly.  I'm starting to think a schema and a table for each common combination would be helpful here for comprehension.

Reply:
I've made a few more updates, let me know what you think.